product-backend/src/test/java/unit/DatabaseConnectorTest.java

```diff
@@ -0,0 +1,25 @@
+package unit;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.kainos.ea.dao.JobDao;
+import org.kainos.ea.db.DatabaseConnector;
+import org.kainos.ea.model.JobRole;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class DatabaseConnectorTest {
```

Comment:
Test is redundant method not used delete it

Reply:
Addressed.